app/controllers/admin/api_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Admin::ApiController < ApplicationController
+  before_filter :admin_only
+  before_filter :check_for_cancel, only: [:create, :update]
+
+  def index
+    @api_keys = if params[:query]
+                  sql_query = "%" + params[:query] + "%"
+                  ApiKey.where("name LIKE ?", sql_query)
+                        .order("name")
+                        .paginate(page: params[:page])
```

Comment:
Trufax, until I saw your comment (must have missed Hound's), I'd never seen the dots left on the previous line in chained method calls in any C-derived languages (though having looked it up, they're also valid in both places in Java and C# too). The Ruby style guide just [lists both options](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains) as equally valid (chickens :D)

My personal opinion about placing them on the previous line is that they will become invisible to the human eye, making it potentially harder to tell that the fragment on the next line is a chained call on the same object, and harder to spot when one of them is missing. But that's probably also because it's what I'm used to! I see that the dot on the previous line is the style used in the very few places where our forebears didn't just leave the line to sprawl off the screen, so I'll adopt this and tell RubyMine to make me do it.